cita-executor/core/src/cita_executive.rs

```diff
@@ -256,50 +255,165 @@ impl<B: DB + 'static> CitaExecutive<B> {
         true
     }
 
-    fn call_amend_data(
-        &mut self,
-        init_gas: U256,
-        value: U256,
-        data: Option<Bytes>,
-    ) -> Result<TxExecResult, ExecutionError> {
+    fn call_amend_data(&mut self, value: U256, data: Option<Bytes>) -> Result<(), ExecutionError> {
```

Comment:
No need to return `TxExecResult` ?

Reply:
Yes, We do not this the result `gas_left`,  because it do not any `gas` in `amend` logic.